### DISCUSSION TITLE
How I create a custom route path?

### DISCUSSION THREAD

**Discussion by ArmandoAmmar** [vercel/next.js#17609] [Help] [answered]
Hi guys, I'm trying to stop to use the file route in my project, I want to create my own route path, how I can create this?

Example:
filepath: 'pages/about.tsx'
route: '/sobre'

PS.: I tried to use rewrites but, this doesn't work how I think.

Thanks in advance!

**Comment by jamesmosier** [comment, marked as answer]
If you are using Next version `9.5` or greater, `rewrites` should be able to solve your problem:

```js
module.exports = {
  async rewrites() {
    return [
      {
        source: '/sobre',
        destination: '/about',
      },
    ]
  },
}
```

What is not working how you think?
